1 explicit pages:
  config/english/config_en.prc

2 implicit pages:
  /c/Panda3D-1.8.1/etc/Config.prc
  /c/Panda3D-1.8.1/etc/Confauto.prc

serverVersion:  sv-1.0.0
PAccount: accountServer from launcher:  
PAccount: default accountServer:  https://pirates.go.com
Initializing quest ladders...
:MagicWordManager(warning): You risk being banned if you abuse this mate!
:MagicWordManager(warning): I will have Jack cut you into pieces! (a.k.a. banned forever)
:DistributedObjectGlobal(warning): ChatManager going online
:DistributedShipLoader(warning): ShipLoader going online
########## startReaderPollTask Pirate
########## stopReaderPollTask Pirate
########## startReaderPollTask Pirate
########## stopReaderPollTask Pirate
New DO:401000001, dclass:PiratesDistrict
New DO:401000002, dclass:DistributedPopulationTracker
getSubTimeList
 sub was None for 0
setEnvironment unnamed 8 {}
getSubTimeList
 sub was None for 8
Traceback (most recent call last):
  File "C:\Panda3D-1.8.1\direct\distributed\ConnectionRepository.py", line 610, in readerPollUntilEmpty
    while self.readerPollOnce():
  File "C:\Panda3D-1.8.1\direct\distributed\ConnectionRepository.py", line 617, in readerPollOnce
    self.handleDatagram(self.private__di)
  File "otp\distributed\OTPClientRepository.py", line 2416, in handleDatagram
    self.handler(msgType, di)
  File "pirates\distributed\PiratesClientRepository.py", line 471, in handleCreateAvatar
    if msgType == CLIENT_CREATE_AVATAR_RESP:
NameError: global name 'CLIENT_CREATE_AVATAR_RESP' is not defined
Traceback (most recent call last):
  File "C:\Panda3D-1.8.1\python\lib\runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "C:\Panda3D-1.8.1\python\lib\runpy.py", line 72, in _run_code
    exec code in run_globals
  File "C:\Users\Tyler\Desktop\POTCOR-src-master\POTCOR-src-master\pirates\piratesbase\PiratesStart.py", line 79, in <module>
    run()
  File "C:\Panda3D-1.8.1\direct\showbase\ShowBase.py", line 2921, in run
    self.taskMgr.run()
  File "C:\Panda3D-1.8.1\direct\task\Task.py", line 502, in run
    self.step()
  File "C:\Panda3D-1.8.1\direct\task\Task.py", line 460, in step
    self.mgr.poll()
  File "C:\Panda3D-1.8.1\direct\distributed\ConnectionRepository.py", line 610, in readerPollUntilEmpty
    while self.readerPollOnce():
  File "C:\Panda3D-1.8.1\direct\distributed\ConnectionRepository.py", line 617, in readerPollOnce
    self.handleDatagram(self.private__di)
  File "otp\distributed\OTPClientRepository.py", line 2416, in handleDatagram
    self.handler(msgType, di)
  File "pirates\distributed\PiratesClientRepository.py", line 471, in handleCreateAvatar
    if msgType == CLIENT_CREATE_AVATAR_RESP:
NameError: global name 'CLIENT_CREATE_AVATAR_RESP' is not defined
